app/src/main/java/org/sopt/study/mvvmstudy/AddActivity.kt

```diff
@@ -0,0 +1,51 @@
+package org.sopt.study.mvvmstudy
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.lifecycle.ViewModelProviders
+import org.sopt.study.mvvmstudy.data.Contact
+import org.sopt.study.mvvmstudy.data.ContactViewModel
+import org.sopt.study.mvvmstudy.databinding.ActivityAddBinding
+
+class AddActivity : AppCompatActivity() {
+
+    private lateinit var contactViewModel: ContactViewModel
```

Comment:
```suggestion
       private val contactViewModel: ContactViewModel by viewModels()
```

ViewModelProivedrs를 쓰지않고 Android KTX를 사용하면 뷰모델 위임 초기화로 더 깔끔하게 생성할 수 있습니다~